### PR TITLE
Added xUnit Runner reference

### PR DESCRIPTION
**Pull request by ranjitCatchinUp**
Fix for #1879.

### CODE REVIEW

**Comment by ranjitCatchinUp** [comment]
Is the CI failure due to mono?

File: src/Nancy.ViewEngines.Razor.Tests.Models/Nancy.ViewEngines.Razor.Tests.Models.csproj [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```
Comment (khellang) [inline src/Nancy.ViewEngines.Razor.Tests.Models/Nancy.ViewEngines.Razor.Tests.Models.csproj]: There's no need to add a runner to this project. It doesn't have any tests :smile:

**Comment by ranjitCatchinUp** [comment]
Still failing. Will try to reset this and add references again. At work now :disappointed:.

**Comment by khellang** [comment]
@ranjitCatchinUp I restarted the build. It's green now :smile:

**Comment by khellang** [comment]
Closes #1879

**Comment by khellang** [comment]
Thanks! :heart:

**Pull request merged**